pinot-common/src/main/java/com/linkedin/pinot/common/data/MetricFieldSpec.java

```diff
@@ -33,10 +33,10 @@
 @SuppressWarnings("unused")
 @JsonIgnoreProperties(ignoreUnknown = true)
 public final class MetricFieldSpec extends FieldSpec {
-  private static final int UNDEFINED_FIELD_SIZE = -1;
+  protected static final int UNDEFINED_METRIC_SIZE = -1;
```

Comment:
Don't export this variable

Reply:
It is protected, and being used only in FieldSpec